packages/typespec-test/test/authoring/generated/typespec-ts/package.json

```diff
@@ -45,8 +45,8 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "^2.5.4",
-    "@azure/abort-controller": "^1.0.0",
+    "@azure/core-lro": "3.0.0-beta.1",
```

Comment:
another question is why are we still using 3.0.0-beta.1 since beta.2 is released several weeks？

Reply:
beta.2 has new changes compared with beta1, we need small adoption effort for it.